editor/src/clj/editor/disk.clj

```diff
@@ -304,19 +304,24 @@
   (disk-availability/push-busy!)
   (future
     (try
-      (let [hook-opts {:output-directory (get bob-options "bundle-output")
+      (let [invoke-bundle-hooks (boolean (some #(= "bundle" %) bob-commands))
```

Comment:
This code may be invoked by the editor script now, and it's not guaranteed we are doing a bundle. I thought about adding a flag to `editor.bob` that controls if we should invoke the bundle hooks, but coming to think of that, we can detect a bundle just by looking at the bob commands.